exchange/targeting_test.go

```diff
@@ -25,18 +25,18 @@ var mockBids = map[openrtb_ext.BidderName][]*openrtb.Bid{
 	openrtb_ext.BidderAppnexus: {{
 		ID:    "losing-bid",
 		ImpID: "some-imp",
-		Price: 0.5,
+		Price: 0.3,
 		CrID:  "1",
 	}, {
 		ID:    "winning-bid",
 		ImpID: "some-imp",
-		Price: 0.7,
+		Price: 0.9,
 		CrID:  "2",
 	}},
 	openrtb_ext.BidderRubicon: {{
 		ID:    "contending-bid",
 		ImpID: "some-imp",
-		Price: 0.6,
+		Price: 0.5,
 		CrID:  "3",
```

Comment:
Why did this values have to change? I noticed that if we roll the price values back to 0.5, 0.7, and 0.6, `TestTargetingCache` found in `exchange/targeting_test.go` fails.

Reply:
@guscarreon  we now can build `hb_pb_cat_dur` values without category and as a result we have less bids back because of deduplication logic. In case of this unit test we have 3 bids: appnexus -> "losing-bid": price 0.5, "winning-bid" 2: price: 0.7, rubicon -> "contending-bid": price 0.6. 
In price_granularity.go -> func getCpmValue(), line 40: 
`roundedCPM := math.Floor(cpm/increment) * increment`
makes price rounding and as result prices turn into these values:
0.6 -> 0.5
0.5 -> 0.5
0.7 -> 0.6
So we have 2 similar bids and after deduplication one of them should be removed. 
I changed values in unit test to make sure all 3 bids are unique and non of them will be removed by deduplication logic.

Reply:
We were discussing this and it does bring to light an issue that having the includeBrandCategories option omitted is changing the default behavior due to the de-duplication logic @Kalypsonika mentioned still being run. We will work on some changes that will allow the test to pass with the original price values, thanks for pointing this out @guscarreon .

Reply:
No problem. Thank you both for addressing my comments so promptly.

Reply:
@guscarreon @jmaynardxandr thank you for your help guys, it's now fixed